Documentation/release-notes/4614.md

```diff
@@ -0,0 +1,8 @@
+* [GitHub PR 4614](https://github.com/xamarin/xamarin-android/pull/4614):
+    Syntax errors in `AndroidManifest.xml` always referred to the generated
+    version of the file (like
+    `obj\Debug\90\android\manifest\AndroidManifest.xml`), even when the syntax
+    error was also present in the hand-written input
+    `Properties\AndroidManifest.xml` file.  Now two errors are provided for any
```

Comment:
I believe that these release notes are now "out of date", as this PR only emits *one* error message, not two.